skills/declarative/People/People/Scripts/DeploymentTemplates/template-with-preexisting-rg.json

```diff
@@ -230,9 +227,7 @@
       "type": "Microsoft.DocumentDB/databaseAccounts/sqlDatabases",
       "apiVersion": "2020-03-01",
       "name": "[concat(variables('cosmosDbAccountName'), '/botstate-db')]",
-      "dependsOn": [
-        "[resourceId('Microsoft.DocumentDB/databaseAccounts', variables('cosmosDbAccountName'))]"
-      ],
+      "dependsOn": ["[resourceId('Microsoft.DocumentDB/databaseAccounts', variables('cosmosDbAccountName'))]"],
```

Comment:
There isn't really a change right? Just going from multi-lines to a single line. Did something update this json?

Reply:
I updated the project based on the template so it would have all the files from the adaptive runtime and base template since it was pretty out of date. This is just from that